src/tds/config.c

```diff
@@ -466,10 +466,10 @@ tds_config_boolean(const char *option, const char *value, TDSLOGIN *login)
 static void
 tds_config_encryption(const char * value, TDSLOGIN * login)
 {
-	TDS_ENCRYPTION_LEVEL lvl = TDS_ENCRYPTION_OFF;
+	TDS_ENCRYPTION_LEVEL lvl = TDS_ENCRYPTION_REQUEST;
 
 	if (!strcasecmp(value, TDS_STR_ENCRYPTION_OFF))
-		;
+		lvl = TDS_ENCRYPTION_OFF;
```

Comment:
The thing that fooled me was that git grep encryption_level did not find tds_alloc_login because the value was being serendipitously set to 0 by the TEST_MALLOC macro using calloc.
I updated my branch, also updating the documentation, and ran this test to make sure I knew exactly what config files were being used by each test:

```
      rm -f /tmp/freetds.check.*
      env FREETDSCONF=$HOME/freetds.conf \
          TDSDUMPCONFIG=/tmp/freetds.check.config.txt TDSDUMP=/tmp/freetds.check.log.txt \
          strace -f -s 128 -o /tmp/freetds.check.strace.txt make check
```

I did not include the patch of @freddy77 yet however. Need to understand how gitlab works better to find it. Stay tuned...